docs/install/install-local-migrate.md

```diff
@@ -28,21 +28,22 @@ Please post questions to the public [Islandora ISLE Google group](https://groups
 
 * You have access to a private git repository in [Github](https://github.com), [Bitbucket](https://bitbucket.org/), [Gitlab](https://gitlab.com), etc.
     * If you do not, please contact your IT department for git resources, or else create an account with one of the above providers.
-    * **WARNING:** Only use **Private** git repositories given the sensitive nature of the configuration files.
-    * **DO NOT** share these git repositories publicly.
+    * **WARNING:** Only use **Private** git repositories given the sensitive nature of the configuration files. **DO NOT** share these git repositories publicly.
 
-* For Windows Users only: You must install [Git for Windows](https://gitforwindows.org/) and use its provided Git BASH command line; this behaves more similarly to LINUX and UNIX environments. (Powershell is not recommended as it is unable to run UNIX commands or execute bash scripts without a moderate degree of customization.) Git for Windows also installs `openssl.exe` which will be needed to generate self-signed SSL certs.
+* **For Microsoft Windows:**
+    * You have installed [Git for Windows](../install/host-software-dependencies.md#windows) and will use its provided "Git BASH" as your command line interface; this behaves more similarly to LINUX and UNIX environments. Git for Windows also installs `openssl.exe` which will be needed to generate self-signed SSL certs. (Note: Powershell is not recommended as it is unable to run UNIX commands or execute bash scripts without a moderate degree of customization.)
+    * Set your text editor to use UNIX style line endings for files. (Text files created on DOS/Windows machines have different line endings than files created on Unix/Linux. DOS uses carriage return and line feed ("\r\n") as a line ending, which Unix uses just line feed ("\n").)
 
 ---
 
 ## Index of Instructions
 
 * Step 0: Copy Production Data to Your Personal Computer
-* Step 1: Edit `/etc/hosts` File
-* Step 2: Setup git for the ISLE Project
+* Step 1: Edit "/etc/hosts" File
+* Step 2: Setup Git for the ISLE Project
 * Step 3: Git Clone the Production Islandora Drupal Site Code
 * Step 4: Edit the .env File to Change to the Local Environment
-* Step 5: Create New Users and Passwords by Editing local.env
+* Step 5: Create New Users and Passwords by Editing "local.env"
```

Comment:
Maybe add " file" to the end of this?

Reply:
agreed. i've added this fix to the next PR.